weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -607,10 +597,11 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
                     }
                     maxLength={24}
                   />{' '}
-                  {props.controlBar === 'editable' && '= '}
+                  {controlBar === 'editable'}
```

Comment:
Seems like this should be removed entirely?

Reply:
lol, yes